### ISSUE TITLE
Build difference between local and action

### ISSUE THREAD

**Issue by The-Code-Monkey** [storybookjs/storybook#18184]
**Describe the bug**
I get this error when building storybook via the chromaui/action 

```
ERR! => Failed to build the preview
ERR! ./node_modules/@aw-web-design/styled-system/dist/esm/core/parser.js 41:44
ERR! Module parse failed: Unexpected token (41:44)
ERR! You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
ERR! |         let styles = {};
ERR! |         let shouldSort = false;
ERR! >         const isCacheDisabled = props.theme?.disableStyledSystemCache;
ERR! |         Object.keys(props).forEach(key => {
ERR! |             if (!config[key]) {
```

but when i run the exact same build command locally it builds no issue.

**To Reproduce**
I have a repo here https://github.com/The-Code-Monkey/TechStack that has the issue on it.

**System**
Environment Info:

  System:
    OS: macOS 12.3.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Binaries:
    Node: 16.15.0 - ~/.nvm/versions/node/v16.15.0/bin/node
    Yarn: 1.22.15 - /usr/local/bin/yarn
    npm: 8.5.5 - ~/.nvm/versions/node/v16.15.0/bin/npm
  Browsers:
    Chrome: 101.0.4951.54
    Safari: 15.4

**Additional context**
I am running on the latest beta of storybook, so im not sure why there is an issue this has been working recently.

**Comment by The-Code-Monkey** [comment]
I have fixed this by downgrading to storybook 6.4 not sure if that was the issue just wanted to update you guys

**Comment by shilman** [comment]
We’re cleaning house! Storybook has changed a lot since this issue was created and we don’t know if it’s still valid. Please open a new issue referencing this one if:
- this is still a problem in SB 7.x
- you can provide a consistent reproduction in 7.x (at https://storybook.new/ or per [repro docs](https://storybook.js.org/docs/react/contribute/how-to-reproduce)).
